packages/eas-cli/src/commands/workflow/run.ts

```diff
@@ -0,0 +1,99 @@
+import chalk from 'chalk';
+import fs from 'node:fs';
+import * as path from 'node:path';
+
+import { getWorkflowRunUrl } from '../../build/utils/url';
+import EasCommand from '../../commandUtils/EasCommand';
+import { WorkflowProjectSourceType } from '../../graphql/generated';
+import { WorkflowRunMutation } from '../../graphql/mutations/WorkflowRunMutation';
+import Log, { link } from '../../log';
+import { getOwnerAccountForProjectIdAsync } from '../../project/projectUtils';
+import { uploadAccountScopedEasJsonAsync } from '../../project/uploadAccountScopedEasJsonAsync';
+import { uploadAccountScopedProjectSourceAsync } from '../../project/uploadAccountScopedProjectSourceAsync';
+
+export default class WorkflowRun extends EasCommand {
+  static override description = 'Run an EAS workflow';
+  static override aliases = ['workflow:run'];
+  static override usage = [chalk`workflow:run {dim [options]}`];
+
+  // TODO(@sjchmiela): Keep command hidden until workflows are live
+  static override hidden = true;
+  static override state = 'beta';
+
+  static override args = [{ name: 'file', description: 'Path to the workflow file to run' }];
+
```

Comment:
```suggestion

static override flags = {
  ...EasNonInteractiveFlag,
}

```

Reply:
Do we need this flag? I removed it on purpose, because the command isn't interactive.